src/utils/dom.js

```diff
@@ -15,6 +15,12 @@ export const states = {
  * Add modal + overlay to DOM
  */
 export const init = (params) => {
+  // Clean up old modals
+  while (getContainer()) {
```

Comment:
@limonte Yeah, that's where I was confused. I didn't see them being removed so I put that there. If I remove the lines now the tests break